src/simple_treeview.rs

```diff
@@ -0,0 +1,98 @@
+//! # TreeView Sample
+//!
+//! This sample demonstrates how to create a TreeView with either a ListStore.
+
+extern crate glib;
+extern crate gtk;
+
+use gtk::prelude::*;
+use gtk::{
+    CellRendererText, Label, ListStore, Orientation, TreeView, TreeViewColumn,
+    Window, WindowPosition, WindowType
+};
+
+fn append_column(tree: &TreeView, id: i32) {
+    let column = TreeViewColumn::new();
+    let cell = CellRendererText::new();
+
+    column.pack_start(&cell, true);
+    // Association of the view's column with the model's `id` column.
+    column.add_attribute(&cell, "text", id);
+    tree.append_column(&column);
+}
+
+fn create_and_fill_model() -> ListStore {
+    // Creation of a model with two rows.
```

Comment:
Any embellishments of the most straightforward form "Create a model..." make the reader spend more cycles on this line than necessary. If that's the intention, fine.